mpt/impl/smtp/cassandra-rabbitmq-object-storage/pom.xml

```diff
@@ -95,7 +95,7 @@
         </dependency>
         <dependency>
             <groupId>${james.groupId}</groupId>
-            <artifactId>james-server-guice-jmap-draft</artifactId>
+            <artifactId>james-server-guice-jmap</artifactId>
```

Comment:
the message for this commit should be: use jmap project everywhere to support both -draft and -rfc8621 implementations